### PR TITLE
Standardizing terms in D.O.C: administrator UI to cover both UIs, administrator

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- We agreed to standardize on Administrator, administrator UI, etc., except for direct reference to the Admin button or other labels "Admin". Not changing any code or blogs.

### Resolves:

[OKTA-161964](https://oktainc.atlassian.net/browse/OKTA-161964)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@jakubvul-okta not ready for review, pretty sure there are broken links to fix.

File: _source/_authentication-guide/implementing-authentication/set-up-authz-server.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ While in the Rules list for an access policy, you can:
 
 ### Rule Usage
 
-Access policy rules are whitelists. If you want to create granular rules, you must first ensure that you have no rules that match "any" of something (for example "Any user"). You can then create specific rules for each specific use case that you do want to support. For example, if you wanted to ensure that only Admin users using the implicit flow were granted access, then you would create a rule specifying that if:
+Access policy rules are whitelists. If you want to create granular rules, you must first ensure that you have no rules that match "any" of something (for example "Any user"). You can then create specific rules for each specific use case that you do want to support. For example, if you wanted to ensure that only Admininstrator users were granted access via an implicit flow, then you would create a rule specifying that if:
```
Comment (mystiberry-okta) [inline _source/_authentication-guide/implementing-authentication/set-up-authz-server.md]: users so close to using was hard to read.

File: _source/_authentication-guide/tokens/validating-access-tokens.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Please note the following:
 
 > Keys used to sign tokens automatically rotate and should always be resolved dynamically against the published JWKS. Your app might fail if you hardcode public keys in your applications. Be sure to include key rollover in your implementation.
 
-> If your application cannot retrieve keys dynamically, the administrator can disable the automatic key rotation in the administration UI, [generate a key credential](/docs/api/resources/apps#generate-new-application-key-credential) and [update the application](/docs/api/resources/apps#update-key-credential-for-application) to use it for signing.
+> If your application cannot retrieve keys dynamically, the administrator can disable the automatic key rotation in the Okta user interface, [generate a key credential](/docs/api/resources/apps#generate-new-application-key-credential) and [update the application](/docs/api/resources/apps#update-key-credential-for-application) to use it for signing.
```
Comment (mystiberry-okta) [inline _source/_authentication-guide/tokens/validating-access-tokens.md]: There were a lot variations on this phrase many of them Okta Admin UI, so I simplified to Okta user interface everywhere (because we're usually trying to disambiguate UI & API, not classic vs. dev console).

**Comment by mystiberry-okta** [comment]
@jakubvul-okta Could you review this?

**Comment by johngronberg-okta** [comment]
I also think "Okta User Interface" is vague, as it could be confused with out End-User UI.

**Comment by mystiberry-okta** [comment]
@jakubvul-okta @johngronberg-okta I don't love it either but we have two UIs our api customers are likely using, dev console and admin. That's too wordy. I assume that people using the UI are familiar with the Admin button. If you have a better suggestion, please send it along. admin UI is not sufficient either, sadly.

**Comment by mystiberry-okta** [comment]
We can spell out the path instead of vague pointing to UI, but then whenever the path changes our doc goes out of date/requires updating. It's actually long past time

File: _source/_docs/api/resources/users.md [hunk 3]
```diff
@@ -3726,11 +3726,11 @@ Okta has a default ambiguous name resolution policy for logins.  Users can login
 For more information about `login`, see [Get User by ID](#get-user-with-id).
 
 ##### Modifying Default Profile Properties
-The only permitted customization of the default profile is changing whether the `firstName` and `lastName` properties are nullable.  You can use the Profile Editor in the Admin UI or the [Schemas API](schemas) to make schema modifications.
+The only permitted customization of the default profile is changing whether the `firstName` and `lastName` properties are nullable.  You can use the Profile Editor in the Okta user interface or the [Schemas API](schemas) to make schema modifications.
 
 #### Custom Profile Properties
 
-User profiles may be extended with custom properties but the property must first be added to the user profile schema before it can be referenced.  You can use the Profile Editor in the Admin UI or the [Schemas API](schemas) to manage schema extensions.
+User profiles may be extended with custom properties but the property must first be added to the user profile schema before it can be referenced.  You can use the Profile Editor in the Okta User Interface or the [Schemas API](schemas) to manage schema extensions.
```
[the remark this answers is not in the document]
Comment (mystiberry-okta) [reply _source/_docs/api/resources/users.md]: yah, I missed that, thanks.

**Comment by mystiberry-okta** [comment]
I can say "Classic UI or Developer Console" but those who don't have dev console never see "Classic UI" AND their dashboard isn't labeled as anything but Dashboard. So, logically, options are:

1. Classic UI or Developer Console
2. administrator's Dashboard or Developer Console
3. Just name the path each time, like Security > API or Applications > Applications or whatever. 
4. Other suggestion, hopefully shorter. These long noun phrases make reading harder.

**Comment by mystiberry-okta** [comment]
@johngronberg-okta and @jakubvul-okta, just confirming that we're going to go with "administrator UI" and put that and UI (instead of user interface, because 21st century) in the style guide. Let me know if you think that's good, and as soon as I get confirmation from InfoDev will update this pr.

**Comment by mystiberry-okta** [comment]
InfoDev says they are standardizing on "admin"  so we'll standardize on "admin UI" (what it was originally in most cases). Ping me if that's not okay.

File: _source/_authentication-guide/implementing-authentication/set-up-authz-server.md (hunk 1 above)
Comment (mystiberry-okta) [reply _source/_authentication-guide/implementing-authentication/set-up-authz-server.md]: fixed typo and put the sentence back the way it was--removed "users" since we want to keep admin and end user as clearly separate as possible.

**Comment by johngronberg-okta** [comment]
"admin UI" SGTM

**Comment by mystiberry-okta** [comment]
@johngronberg-okta  I misspoke--InfoDev doesn't need us to use "admin". It's administrator now, but easy to change back to admin. I have this intuition that "admin" seems to exclude the dev console more than administrator UI, but that might be just me. Let me know what you think (and @jakubvul too!)

**Comment by mystiberry-okta** [comment]
@jakubvul-okta I think this is ready for review again!
